openvdb_points/unittest/TestAttributeArray.cc

```diff
@@ -297,6 +299,22 @@ TestAttributeArray::testAttributeArray()
 
 
 void
+TestAttributeArray::testAttributeArrayCasting()
```

Comment:
This shouldn't be a new unit test, let's merge this with the existing attributeArray test.